spec/lib/pdf_filler/id40_pdf_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PdfFiller::Id40Pdf do
+  include PdfSpecHelper
+
+  let!(:intake) {
+    create(:state_file_id_intake,
+           :single_filer_with_json, # includes phone number data
+           primary_esigned: "yes",
+           primary_esigned_at: DateTime.now)
+  }
+  let(:submission) { create :efile_submission, tax_return: nil, data_source: intake }
+  let(:pdf) { described_class.new(submission) }
+
+  describe '#hash_for_pdf' do
+    let(:pdf_fields) { filled_in_values(described_class.new(submission).output_file.path) }
+
+    it 'uses field names that exist in the pdf' do
+      missing_fields = pdf.hash_for_pdf.keys.map(&:to_s) - pdf_fields.keys
+      expect(missing_fields).to eq([])
+    end
+
+    context "when filer signed submission agreement" do
+      it 'sets signature date field to the correct value' do
+        expect(pdf_fields["DateSign 2"]).to eq DateTime.now.strftime("%m-%d-%Y")
```

Comment:
@anisharamnani  This test expectation failed for me just now on CircleCI, while working over on the NJ side of things. It started failing right after 9pm Pacific. So I suspect that someplace there is a mismatch of Eastern vs Pacific time?

```
 1) PdfFiller::Id40Pdf#hash_for_pdf when filer signed submission agreement sets signature date field to the correct value
     Failure/Error: expect(pdf_fields["DateSign 2"]).to eq DateTime.now.strftime("%m-%d-%Y")
     
       expected: "11-01-2024"
            got: "10-31-2024"
     
       (compared using ==)
     # ./spec/lib/pdf_filler/id40_pdf_spec.rb:25:in `block (4 levels) in <main>'
     # ./spec/support/locale.rb:4:in `block (2 levels) in <main>'
```

https://app.circleci.com/pipelines/github/codeforamerica/vita-min/25449/workflows/a21fe13e-af76-4c14-a717-2a570e685d16/jobs/69203